packages/react-day-picker/src/components/Table/utils/getWeeks.ts

```diff
@@ -14,51 +13,44 @@ import { getOutsideEndDays } from './getOutsideEndDays';
 import { getOutsideStartDays } from './getOutsideStartDays';
 
 /**
- * The weeks belonging to a month. Each key of the returned object is the
- * week number of the year.
+ * The date-fns week and the dates for that week
  */
-export type MonthWeeks = { [weeknumber: string]: Date[] };
-
+export type MonthWeek = { weekNumber: number; dates: Date[] };
```

Comment:
Since this type and the `getWeeks` function were previously publicly exported, this change is a breaking change, but I assumed that is ok because we are still in beta.

Reply:
I think it is a pretty nice solution. I wasn't sure at first about the `Object`, which i think I choose for performance reasons.